tests/integration/test_polarion.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+import pytest
+from click.testing import CliRunner
+from fmf import Tree
+
+import tmt.cli
+from tmt.identifier import ID_KEY
+
+from .test_nitrate import TEST_DIR, Base
+
+
+@pytest.mark.skip(reason="Only works locally for now")
+class PolarionExport(Base):
+    EXAMPLES = TEST_DIR / "data" / "polarion"
+
+    def test_create(self):
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertNotIn(ID_KEY, fmf_node.data)
+
+        os.chdir(self.tmpdir / "new_testcase")
+        runner = CliRunner()
+        self.runner_output = runner.invoke(tmt.cli.main, [
+            "test", "export", "--how", "polarion", "--project-id",
+            "RHIVOS", "--create", "--general", "."])
+        # Reload the node data to see if it appears there
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertIn(ID_KEY, fmf_node.data)
+
+    def test_create_dryrun(self):
+        fmf_node_before = Tree(self.tmpdir).find("/new_testcase")
+        self.assertNotIn(ID_KEY, fmf_node_before.data)
+
+        os.chdir(self.tmpdir / "new_testcase")
+        runner = CliRunner()
+        self.runner_output = runner.invoke(
+            tmt.cli.main,
+            ["test", "export", "--how", "polarion", "--create", "--dry", "."],
+            catch_exceptions=False)
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertNotIn(ID_KEY, fmf_node.data)
+        self.assertEqual(fmf_node_before.data, fmf_node.data)
+        self.assertIn(
+            "title: tmt /new_testcase - This i",
+            self.runner_output.output)
+
+    def test_existing(self):
+        fmf_node = Tree(self.tmpdir).find("/existing_testcase")
+        self.assertEqual(fmf_node.data["extra-nitrate"], "TC#0609686")
+
+        os.chdir(self.tmpdir / "existing_testcase")
+        runner = CliRunner()
+        self.runner_output = runner.invoke(tmt.cli.main, [
+            "test", "export", "--how", "polarion", "--project-id",
+            "RHIVOS", "--create", "--general", "."])
```

Comment:
not at the moment, I'm thinking this can then automatically link the test case to requirement/s, but didn't want to do it just yet as requirements are a mess at this moment